### PR TITLE
Fix: Spacing of inputs and outputs in transaction section

### PR DESCRIPTION
**Pull request by abbasabid**
# Description of change

- Added left border for all the sections of inputs and outputs.
- Fix spacing between different sections.

Resolves https://github.com/iotaledger/explorer/issues/576 https://github.com/iotaledger/explorer/issues/581

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

### CODE REVIEW

File: client/src/app/components/stardust/TransactionPayload.scss [hunk 1]
```diff
@@ -83,7 +83,7 @@
     }
   }
 
-  .transaction-from.card--content .card--content__output {
+  .transaction.card--content .card--content__output {
```
Comment (brancoder) [inline client/src/app/components/stardust/TransactionPayload.scss:86]: Maybe we could change the `transaction` class name to  `transaction-payload_outputs` so its in line with BEM naming convention.

**Review by brancoder — CHANGES_REQUESTED** [review]
We can add the border and spacing changes to unlocks.
image

File: client/src/app/components/stardust/Unlocks.tsx [hunk 2]
```diff
@@ -11,10 +11,11 @@ const Unlocks: React.FC<IUnlocksProps> = ({ unlocks }) => {
     const [isExpanded, setIsExpanded] = useState(false);
 
     return (
-        <div className="card--content__output ">
+        <div className="card--content__output">
             <div
                 onClick={() => setIsExpanded(!isExpanded)}
                 className="card--value card-header--wrapper"
+                style={{ height: "32px" }}
```
Comment (brancoder) [inline client/src/app/components/stardust/Unlocks.tsx:18]: Can we try to avoid using hardcoded style values?

**Review by brancoder — CHANGES_REQUESTED** [review]
- Align the top and bottom margins so they have the same value

- Reduce the spacing between the rows as per #576  
> Spacing should be equal. Preferably the smaller of the two.

outputs

**Comment by msarcev** [comment]
image
Transaction payload now broken on mainnet...

**Comment by abbasabid** [comment]
> image Transaction payload now broken on mainnet...

Fixed.

**Review by brancoder — APPROVED** [review]
LGTM

**Pull request merged**